docs/feature_oled_driver.md

```diff
@@ -166,6 +166,38 @@ bool oled_task_user(void) {
 #endif
 ```
 
+Render a message before booting into bootloader mode.
+```c
+void oled_render_boot(void) {
+    oled_clear();
+    for (int i = 0; i < 16; i++) {
+        oled_set_cursor(0, i);
+        oled_write_P(PSTR("BOOT "), false);
+    }
+
+    oled_render_dirty(true);
+}
+
+bool reboot = false;
+
+bool uint16_t keycode, keyrecord_t *record) {
+    if (record->event.pressed) {
+  
+        // Display a special message prior to rebooting...
+        if (keycode == QK_BOOT) {
+           reboot = true;
+        }
+    }
+
+    return true;
+}
+
+void shutdown_user(void) {
+    oled_render_boot();
```

Comment:
And that's what I get for chopping things into smaller pieces .. I'll add the missing if in shortly.